core/net/ip/uip-debug.c

```diff
@@ -75,6 +75,10 @@ void
 uip_debug_lladdr_print(const uip_lladdr_t *addr)
 {
   unsigned int i;
+  if(addr == NULL) {
+    printf("(NULL LL addr)");
```

Comment:
That happens when you blindly copy&paste the code from the function uip_debug_ipaddr_print() above :) I'll fix both and also move the check after the local declaration in uip_debug_ipaddr_print()